openmetadata-ui/src/main/resources/ui/src/components/Ingestion/Ingestion.test.tsx

```diff
@@ -0,0 +1,223 @@
+    const { container } = render(
+      <Ingestion
+        addIngestion={mockFunction}
+        deleteIngestion={mockDeleteIngestion}
+        ingestionList={mockIngestionWorkFlow.data.data as IngestionData[]}
+        paging={mockPaging}
+        pagingHandler={mockPaginghandler}
+        serviceList={[]}
+        triggerIngestion={mockTriggerIngestion}
+        updateIngestion={mockFunction}
+      />,
+      {
+        wrapper: MemoryRouter,
+      }
+    );
+
+    const ingestionTable = await findByTestId(container, 'ingestion-table');
+    const tableHeaderContainer = await findByTestId(container, 'table-header');
+    const runButton = await findByTestId(container, 'run');
+    const editButton = await findByTestId(container, 'edit');
+    const deleteButton = await findByTestId(container, 'delete');
+    const tableHeaders: string[] = [];
+
+    tableHeaderContainer.childNodes.forEach(
+      (node) => node.textContent && tableHeaders.push(node.textContent)
+    );
+
+    expect(ingestionTable).toBeInTheDocument();
+    expect(tableHeaderContainer).toBeInTheDocument();
+    expect(tableHeaders.length).toBe(6);
+    expect(tableHeaders).toStrictEqual([
+      'Name',
+      'Type',
+      'Service',
+      'Schedule',
+      'Recent Runs',
+      'Actions',
+    ]);
+    expect(runButton).toBeInTheDocument();
+    expect(editButton).toBeInTheDocument();
+    expect(deleteButton).toBeInTheDocument();
+  });
+
+  it('Pagination should be render if paging is provided', async () => {
+    const mockPagingAfter = {
+      after: 'afterKey',
+      before: 'beforeKey',
+    };
+    const { container } = render(
+      <Ingestion
+        addIngestion={mockFunction}
+        deleteIngestion={mockDeleteIngestion}
+        ingestionList={mockIngestionWorkFlow.data.data as IngestionData[]}
+        paging={mockPagingAfter}
+        pagingHandler={mockPaginghandler}
+        serviceList={[]}
+        triggerIngestion={mockTriggerIngestion}
+        updateIngestion={mockFunction}
+      />,
+      {
+        wrapper: MemoryRouter,
+      }
+    );
+
+    const nextPrevious = await findByText(container, /NextPrevious/i);
+
+    expect(nextPrevious).toBeInTheDocument();
+  });
+
+  it('CTA should work', async () => {
+    const mockPagingAfter = {
+      after: 'afterKey',
+      before: 'beforeKey',
+    };
+
+    const { container } = render(
+      <Ingestion
+        addIngestion={mockFunction}
+        deleteIngestion={mockDeleteIngestion}
+        ingestionList={mockIngestionWorkFlow.data.data as IngestionData[]}
+        paging={mockPagingAfter}
+        pagingHandler={mockPaginghandler}
+        serviceList={[]}
+        triggerIngestion={mockTriggerIngestion}
+        updateIngestion={mockFunction}
+      />,
+      {
+        wrapper: MemoryRouter,
+      }
+    );
+
+    // on click of add ingestion
+    const addIngestionButton = await findByTestId(
+      container,
+      'add-new-user-button'
```

Comment:
This should be `add-new-ingestion-button`

Reply:
Correct, updated.